students/mroz_sebastian/lesson_01_python_basics/l1_snakify/Apple_sharing.py

```diff
@@ -0,0 +1,4 @@
+n = int(input())
```

Comment:
file names are generally modules in python. Module name should be lowercase separated by underscores
students/mroz_sebastian/lesson_01_python_basics/l1_snakify/apple_sharing.py